Makefile

```diff
@@ -205,7 +205,7 @@ ifeq ($(KOPIA_EXE),)
 robustness-tests: dist-binary
 	FIO_DOCKER_IMAGE=$(FIO_DOCKER_TAG) \
 	KOPIA_EXE=$(KOPIA_INTEGRATION_EXE) \
-	$(GO_TEST) -count=1 github.com/kopia/kopia/tests/robustness/robustness_test $(TEST_FLAGS)
+	$(GO_TEST) -v -count=1 github.com/kopia/kopia/tests/robustness/robustness_test $(TEST_FLAGS)
```

Comment:
Is this required?
I'm not sure if this was omitted on purpose to reduce logs.

Reply:
Let's remove it from the default invocation and rely on `$(TEST_FLAGS)`.
The rationale is that we need to reduce the output verbosity for jobs. Otherwise the logs become too large to display in the Shippable console, also the jobs end up running longer and running into timeouts.